project/templates/homePage.php

```diff
@@ -44,32 +44,16 @@
         </div>
         <div id="runFormHolder">
             <form action="../controller/newRunController.php" method="POST">
-                <br><br><span> Distance </span>
-                <input type="text" name="distanceTraveled" required>
-                <br><br><span> Time of start </span>
+                <br><br><span> Date of run </span>
                 <input type="date" name="dateOfRun">
                 <br><br><span> Time of start </span>
                 <input type="time" name="startAtTime">
                 <br><br><span> Time of end </span>
                 <input type="time" name="finishAtTime">
```

Comment:
Why did you added new fields, deleted old ones, but not updated newRunController.php?

Reply:
Fixed